### PR TITLE
convert WrapperDesign to a function component

### PR DESCRIPTION
**Pull request by dimitropoulos**
closes INS-1389

This PR converts the WrapperDesign component to a function component and tries to change as little else as possible.  3 of the 4 sections were pretty straightforward, but the Editor one had some fidgeting that was necessary to keep the debounce behavior intact.  There are no intended behavioral changes as a result of this PR.

I encourage any reviewer to read commit by commit.  With the exception of the very last commit, which contains a small fix, I tried to keep the scope of each commit constrained to what's in the commit message.

### CODE REVIEW

File: packages/insomnia-app/app/ui/components/wrapper-design.tsx [hunk 1]
```diff
@@ -31,291 +30,285 @@ const EmptySpaceHelper = styled.div({
...
 
-    return (
+  return (
+    <div id="swagger-ui-wrapper">
       <ErrorBoundary
         invalidationKey={activeApiSpec.contents}
         renderError={() => (
           <div className="text-left margin pad">
-            <h4>An error occurred while trying to render your spec's navigation.</h4>
+            <h3>An error occurred while trying to render Swagger UI</h3>
```
Comment (gatzjames) [inline packages/insomnia-app/app/ui/components/wrapper-design.tsx:194]: Should we leak about Swagger UI in the UI?
I think the previous wording was more general

Comment (dimitropoulos) [reply packages/insomnia-app/app/ui/components/wrapper-design.tsx:194]: I changed the wording???

Comment (dimitropoulos) [reply packages/insomnia-app/app/ui/components/wrapper-design.tsx:194]: ok, @gatzjames just look a little closer.  wow, had me worried there!  this is just git being strange.  I didn't change any wording anywhere (and don't want to in the context of the scope of this PR).  It's just git being confused about lines.  This is just very similar to a different message we show, which is also preserved.

Comment (gatzjames) [reply packages/insomnia-app/app/ui/components/wrapper-design.tsx:194]: 🙈 That was so close though 😂

**Review by gatzjames — APPROVED** [review]
Awesome! LGTM!

**Pull request merged**